examples/kubevirt-disk-uploader-tekton.yaml

```diff
@@ -182,8 +185,14 @@ spec:
         secretKeyRef:
           name: kubevirt-disk-uploader-credentials-tekton
           key: registryPassword
+    - name: VM_NAMESPACE
```

Comment:
in this example the env variable `VM_NAMESPACE` will be always used instead of `vmnamespace` param. Is it expected behaviour?

Reply:
Nope, and I think it would confuse users. Let's just stick to `VM_NAMESPACE` env var.